### ISSUE TITLE
1.4.2 fails to drain nodes

### ISSUE THREAD

**Issue by someone-stole-my-name** [TwiN/aws-eks-asg-rolling-update-handler#27]
Newer versions require a context in the helper:

https://github.com/kubernetes/kubernetes/pull/105297

1.4.0 -> https://github.com/TwiN/aws-eks-asg-rolling-update-handler/blob/v1.4.0/vendor/k8s.io/kubectl/pkg/drain/default.go#L51
1.4.2 -> https://github.com/TwiN/aws-eks-asg-rolling-update-handler/blob/v1.4.2/vendor/k8s.io/kubectl/pkg/drain/default.go#L53

```
2022/08/11 06:36:10 [xx][i-1] Updated nodes have enough resources available
2022/08/11 06:36:10 [xx][i-1] Draining node
2022/08/11 06:36:10 [ip-1][DRAINER] Failed to cordon node: RunCordonOrUncordon error: drainer.Ctx can't be nil
2022/08/11 06:36:10 [xx][i-1] Skipping because ran into error while draining node: RunCordonOrUncordon error: drainer.Ctx can't be nil
```

**Comment by TwiN** [comment]
@someone-stole-my-name Thank you for bringing it up!

**Comment by TwiN** [comment]
Released in [v1.4.3](https://github.com/TwiN/aws-eks-asg-rolling-update-handler/releases/tag/v1.4.3).

Any chance you could give it a try and let me know? I've added a test to ensure it doesn't happen again, but just in case I missed something else..
